scripts/google_research/open-covid-19/open_covid_19_data.mcf

```diff
@@ -0,0 +1,934 @@
+
+Node: dcid:PolicyExtent_Legislation_COVID19Pandemic_GovernmentOrganization_ContactTracing
+typeOf: dcs:StatisticalVariable
+populationType: schema:Legislation
+legislationPassedBy: schema:GovernmentOrganization
+category: dcs:COVID19Pandemic
+policyType: dcs:ContactTracing
+measuredProperty: dcs:policyExtent
+
+Node: dcid:Amount_Legislation_COVID19Pandemic_GovernmentOrganization_ShortTermSpending_HealthcareExpenditure
+typeOf: dcs:StatisticalVariable
+populationType: schema:Legislation
+category: dcs:COVID19Pandemic
+legislationPassedBy: schema:GovernmentOrganization
+policyType: dcs:ShortTermSpending
+expenditure: dcs:HealthcareExpenditure
+measuredProperty: schema:amount
+
+Node: dcid:Amount_Legislation_COVID19Pandemic_GovernmentOrganization_ShortTermSpending_VaccineExpenditure
+typeOf: dcs:StatisticalVariable
+populationType: schema:Legislation
+category: dcs:COVID19Pandemic
+legislationPassedBy: schema:GovernmentOrganization
+policyType: dcs:ShortTermSpending
+expenditure: dcs:VaccineExpenditure
+measuredProperty: schema:amount
+
+Node: dcid:Amount_Legislation_COVID19Pandemic_GovernmentOrganization_ShortTermSpending_EconomicStimulusExpenditure
+typeOf: dcs:StatisticalVariable
+populationType: schema:Legislation
+category: dcs:COVID19Pandemic
+legislationPassedBy: schema:GovernmentOrganization
+policyType: dcs:ShortTermSpending
+expenditure: dcs:EconomicStimulusExpenditure
+measuredProperty: schema:amount
+
+Node: dcid:Amount_Legislation_COVID19Pandemic_GovernmentOrganization_ShortTermSpending_InternationalAidExpenditure
+typeOf: dcs:StatisticalVariable
+populationType: schema:Legislation
+category: dcs:COVID19Pandemic
+legislationPassedBy: schema:GovernmentOrganization
+policyType: dcs:ShortTermSpending
+expenditure: dcs:InternationalAidExpenditure
+measuredProperty: schema:amount
+
+Node: dcid:Covid19StringencyIndex_Legislation_COVID19Pandemic_GovernmentOrganization
+typeOf: dcs:StatisticalVariable
+populationType: schema:Legislation
+legislationPassedBy: schema:GovernmentOrganization
+category: dcs:COVID19Pandemic
+measuredProperty: dcs:covid19StringencyIndex
+
+Node: dcid:Covid19ResponseIndex_Legislation_COVID19Pandemic_GovernmentOrganization
+typeOf: dcs:StatisticalVariable
+populationType: schema:Legislation
+legislationPassedBy: schema:GovernmentOrganization
+category: dcs:COVID19Pandemic
+measuredProperty: dcs:covid19ResponseIndex
+
+Node: dcid:Covid19ContainmentAndHealthIndex_Legislation_COVID19Pandemic_GovernmentOrganization
+typeOf: dcs:StatisticalVariable
+populationType: schema:Legislation
+legislationPassedBy: schema:GovernmentOrganization
+category: dcs:COVID19Pandemic
+measuredProperty: dcs:covid19ContainmentAndHealthIndex
+
+Node: dcid:Covid19EconomicSupportIndex_Legislation_COVID19Pandemic_GovernmentOrganization
+typeOf: dcs:StatisticalVariable
+populationType: schema:Legislation
+legislationPassedBy: schema:GovernmentOrganization
+category: dcs:COVID19Pandemic
+measuredProperty: dcs:covid19EconomicSupportIndex
```

Comment:
@intrepiditee should these go under schema/ ?

Reply:
I removed the enums and only kept the statvars